pkg/cmd/build/buildkit.go

```diff
@@ -131,17 +131,17 @@ func getSolveOpt(buildOptions *types.BuildOptions) (*client.SolveOpt, error) {
 	}
 
 	if buildOptions.ExportCache != "" {
-		mode := "min"
+		exportType := "inline"
```

Comment:
There should be a test added for this. It looks like we should be asserting specific build options being set as a default when export cache is an empty string?

What's the best way to follow-up on that?

Reply:
@jLopezbarb is going to send a follow up PR with an e2e test too